extensions/amp-form/0.1/form-validators.js

```diff
@@ -93,6 +96,18 @@ export class FormValidator {
      * @private {boolean|null}
      */
     this.formValidity_ = null;
+
+    /** @private {string} */
+    this.uniqueFormId_ = this.form.id
+      ? this.form.id
+      : String(Date.now() + Math.floor(Math.random() * 100));
+
+    /**
+     * Counter used to create a unique id for every validation message
+     * to be used with `aria-describedby`.
+     * @protected {number}
```

Comment:
nit: let's move these new properties and methods to `AbstractCustomValidator` since they're only consumed by that subclass, and make them  `@private`.